src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -181,18 +178,25 @@ const nextUrl = ref<string | null>(null)
 const filteredDataPlaneType = ref('All')
 const pageOffset = ref(props.offset)
 const dataPlaneOverview = ref<DataPlaneOverview | null>(null)
-
 const isMultiZoneMode = computed(() => store.getters['config/getMulticlusterStatus'])
 const dataplaneWizardRoute = computed(() => ({ name: store.getters['config/getEnvironment'] === 'universal' ? 'universal-dataplane' : 'kubernetes-dataplane' }))
+
+const isGateway = (item: DataPlaneOverview) => {
+  return typeof item.dataplane.networking.gateway !== 'undefined'
```

Comment:
**Suggestion**: Is there any benefit in not comparing the values directly?

```suggestion
  return item.dataplane.networking.gateway !== undefined
```

Reply:
Half habit, half consistency with things like `typeof item.method === 'function'`